spec/index.bs

```diff
@@ -6475,8 +6490,23 @@ following members:
 
                                     :  {{GPUBindGroupLayoutEntry/externalTexture}}
                                     ::
-                                        - |resource| is a {{GPUExternalTexture}}.
+                                        - |resource| is either a {{GPUExternalTexture}} or a {{GPUTextureView}}.
                                         - |resource| is [$valid to use with$] |this|.
+                                        - If |resource| is a:
+                                            <dl class=switch>
+                                                : {{GPUTextureView}}
+                                                ::
+                                                    - |resource|.{{GPUTextureView/[[descriptor]]}}.{{GPUTextureViewDescriptor/usage}}
+                                                        must include {{GPUTextureUsage/TEXTURE_BINDING}}.
+                                                    - |resource|.{{GPUTextureView/[[descriptor]]}}.{{GPUTextureViewDescriptor/dimension}}
+                                                        must be {{GPUTextureViewDimension/"2d"}}.
+                                                    - |resource|.{{GPUTextureView/[[descriptor]]}}.{{GPUTextureViewDescriptor/mipLevelCount}}
+                                                        must be 1.
+                                                    - |resource|.{{GPUTextureView/[[descriptor]]}}.{{GPUTextureViewDescriptor/format}}
+                                                        must be one of the [=Supported context formats=].
```

Comment:
I think we should duplicate the list here. If we add more canvas formats we don't necessarily want to allow them here. We probably wouldn't notice we were changing the behavior of externalTexture since the name doesn't seem like it would.
```suggestion
                                                        must be {{GPUTextureFormat/"rgba8unorm"}},
                                                        {{GPUTextureFormat/"bgra8unorm"}}, or
                                                        {{GPUTextuerFormat/"rgba16float"}}.
```